package.json

```diff
@@ -7,7 +7,10 @@
   "author": "ESPHome maintainers",
   "license": "Apache-2.0",
   "scripts": {
-    "prepublishOnly": "script/build"
```

Comment:
Do not remove prepublishOnly. That is used to ensure we have a build before publishing

Reply:
Google it?